rollup.module-exports.mjs

```diff
@@ -57,11 +57,12 @@ export default {
   // ChannelSettings
   ChannelSettings: 'src/modules/ChannelSettings/index.tsx',
   'ChannelSettings/context': 'src/modules/ChannelSettings/context/ChannelSettingsProvider.tsx',
+  'ChannelSettings/hooks/useMenuList': 'src/modules/ChannelSettings/components/ChannelSettingsUI/hooks/useMenuItems.tsx',
   'ChannelSettings/components/ChannelProfile': 'src/modules/ChannelSettings/components/ChannelProfile/index.tsx',
   'ChannelSettings/components/ChannelSettingsUI': 'src/modules/ChannelSettings/components/ChannelSettingsUI/index.tsx',
   'ChannelSettings/components/ChannelSettingsHeader': 'src/modules/ChannelSettings/components/ChannelSettingsUI/ChannelSettingsHeader.tsx',
   'ChannelSettings/components/ChannelSettingMenuList': 'src/modules/ChannelSettings/components/ChannelSettingsUI/MenuListByRole.tsx',
-  'ChannelSettings/hooks/useMenuList': 'src/modules/ChannelSettings/components/ChannelSettingsUI/hooks/useMenuItems.tsx',
+  'ChannelSettings/components/ChannelSettingsMenuItem': 'src/modules/ChannelSettings/components/ChannelSettingsUI/MenuItem.tsx',
```

Comment:
please confirm the exporting path name @AhyoungRyu 
```
import { MenuItem } from '@sendbird/uikit-react/ChannelSettings/components/ChannelSettingsMenuItem'
```

Reply:
Yeah looks good to me.